app/src/main/res/layout/status_widget.xml

```diff
@@ -0,0 +1,16 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    style="@style/Widget.Traccarclientandroid.AppWidget.Container"
+    android:background="@android:color/transparent"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:theme="@style/AppTheme.AppWidgetContainer">
```

Comment:
Is the theme required? Looks like it adds a lot of code.

Reply:
Theme support was generated by Android Studio from the template.
According to the [documentation](https://developer.android.google.cn/guide/topics/appwidgets/enhance?hl=en) it mainly affects newer versions of Android (12+).
If it isn't needed, I can clean it up.